pkg/commands/logtail/root.go

```diff
@@ -446,7 +447,11 @@ func (c *RootCommand) printLogs(out io.Writer, logs []Log) {
 		filtered := filterStream(c.cfg.stream, logs)
 
 		for _, l := range filtered {
-			fmt.Fprintln(out, l.String())
+			if c.cfg.printTimestamps {
+				fmt.Fprintf(out, "%s | %s\n", l.RequestStartFromRaw().UTC().Format(time.RFC3339), l.String())
+			} else {
+				fmt.Fprintln(out, l.String())
+			}
```

Comment:
I wasn't sure if it was better to format the string here, or add a `StringWithTimestamp` method on the `Log`.  Happy to do whichever you think is best.

Reply:
Since `Log` is only used in this module, and the output format is not really part of that structure, I have no objections to it being done here. If Go had a ternary operator this would be a good place to use it, but...

Maybe one small improvement would be to just declare a string variable that is either empty or contains the formatted timestamp prefix (with separator) and then call Fprintf only once, including both that string and `l.String()`.

Reply:
Variation on this which should limit formatting & allocations done in 681638d